### DISCUSSION TITLE
Devnet Redeployment 2024-08-12

### DISCUSSION THREAD

**Discussion by mlegner** [MystenLabs/walrus-docs#103] [Announcements]
Dear builders,

Thank you very much for trying out Walrus and providing feedback at this early stage. Based on your valuable input, we have made several improvements to the Walrus system.

We will update our Devnet on Monday, 2024-08-12, which includes deploying new contracts and binaries and wiping all data currently stored on Walrus. This means that you will have to download the new binary and configuration and re-upload any previously stored data.

This also affects all deployed Walrus Sites, which you will have to re-deploy. Further details and information about changes ~~will be provided in our docs, the draft is already available [here](https://mystenlabs.github.io/walrus-docs/pr-preview/pr-102/blog/02_devnet_update.html)~~ are available [here](https://docs.walrus.site/blog/02_devnet_update.html).

During the redeployment, Walrus and Walrus Sites will be unavailable, including our own sites like <https://docs.walrus.site>.

We will comment below this announcement when we start with the upgrade process and again when the
re-deployed system is up.

Thank you very much for your patience and please continue to provide feedback and feature requests.

With best wishes,
The Walrus team

**Comment by giac-mysten** [comment]
⚠️ Hello everyone, the Walrus redeployment is about to begin!
Walrus and Walrus Sites may be unavailable until further notice.

**Comment by karlwuest** [comment]
Hello everyone,
Walrus has been successfully redeployed. See [here](https://docs.walrus.site/blog/02_devnet_update.html) for the changes that come with this upgrade.
